index.js

```diff
@@ -1,29 +1,25 @@
 'use strict';
-var net = require('net');
-var onetime = require('onetime');
+const net = require('net');
 
-module.exports = function (port, opts, cb) {
-	if (typeof opts !== 'object') {
-		cb = opts;
-		opts = {};
-	}
+module.exports = (port, opts) => {
+	opts = Object.assign({}, opts);
```

Comment:
I mostly do it so that I don't have to care about checking if `opts` exists or not later on.